node/src/types/block.rs

```diff
@@ -978,13 +1006,17 @@ impl Block {
         let state_root_hash = Digest::random(rng);
         let finalized_block = FinalizedBlock::random(rng);
         let parent_seed = Digest::random(rng);
+        let next_era_validator_weights = match finalized_block.era_end {
+            Some(_) => Some(BTreeMap::<PublicKey, U512>::default()),
```

Comment:
I'd rather use a non-empty random map. An empty set of validators makes no sense.

Reply:
`next_era_validator_weights` could naturally contain `finalized_block.proposer`, and you could generate some other ones along with that.

Reply:
It doesn't have to contain the proposer, though; they could have unbonded, and be out in the next era. Would be good to test both cases. (Although probably at the moment we don't have any tests where that matters.)

Reply:
Removed default scenario